src/shared/scrapers/US/KY/index.js

```diff
@@ -21,7 +21,7 @@ const scraper = {
       name: 'Jordan Holt',
       email: 'jordholt@gmail.com',
       github: 'Jord-Holt',
-      country: 'USA',
+      country: 'iso1:US',
```

Comment:
Is this legit?

Reply:
In maintainers we could change all those back to USA I think. Right now the search-replace was safe and consistent and I preferred the safety of that.